src/utils/logging.cc

```diff
@@ -30,11 +30,55 @@
 #include <iostream>
 #include <thread>
 
+#ifdef _WIN32
+#include <windows.h>
+#endif
+
 namespace lczero {
 
 namespace {
 size_t kBufferSizeLines = 200;
 const char* kStderrFilename = "<stderr>";
+
+#ifdef _WIN32
```

Comment:
It's not nice to have platform-specific code under ifdefs, especially given it's not that related to the core purpose of logging.
Maybe add this function to `filesystem.h` (and corresponding `.win32.cc`/ `.posix.cc`)?
It's not that filesystem related but adding one more set of files is probably an overkill. Or maybe `filesystem.*` may be renamed to something like `os.*`, but it doesn't look too good either.